style/middleware/_middleware.filepicker.scss

```diff
@@ -0,0 +1,244 @@
+/* ==========================================================================
+   #FILEPICKER - Core Moodle Dialog box to be used for v1
+   @TO-DO: Remove when Nightingale has dialog-box styling in future versions
+   ========================================================================== */
+
+// Grid breakpoints
+//
+// Define the minimum dimensions at which your layout will change,
+// adapting to different screen sizes, for use in media queries.
+
+$grid-breakpoints: (
+        xs: 0,
```

Comment:
Are these mapped to the nightingale break points?

Reply:
No, our nightingale breakpoints are called `mq-breakpoints`. I have kept the two separate now to avoid any confusion. 
These are taken from `variables.scss` and these would only be used for core Moodle file-picker/modal dialog files that we have imported. Also the values of breakpoints from our nightingale is different than the one Moodle demands so all the more had to copy in Moodle's grid breakpoints to get the exact styling!